antarest/study/common/studystorage.py

```diff
@@ -17,13 +17,20 @@
 from antarest.core.exceptions import StudyNotFoundError
 from antarest.core.model import JSON
 from antarest.core.requests import RequestParameters
+from antarest.core.serialization import AntaresBaseModel
 from antarest.study.model import Study, StudyMetadataDTO, StudyMetadataPatchDTO, StudySimResultDTO
 from antarest.study.storage.rawstudy.model.filesystem.config.model import FileStudyTreeConfigDTO
 from antarest.study.storage.rawstudy.model.filesystem.factory import FileStudy
 
 T = t.TypeVar("T", bound=Study)
 
 
+class OriginalFile(AntaresBaseModel):
```

Comment:
More a thought, not a must-do: for such simple classes that are never serialized, I would propose to favor plain dataclasses. I think it makes the intent more clear: it's just a simple struct used in-memory.